getting_started.md

```diff
@@ -0,0 +1,109 @@
+
+### Coding your submission
+#TODO(kasimbeg): If we want use this submission_template script where should we store it in the repo?
+
+You can find examples of sumbission modules under `algorithmic-efficiency/baselines` and `algorithmic-efficiency/reference_algorithms`. \
+A submission for the external ruleset will consist of a submission module and a tuning search space definition.
+1. Clone the submission_template.py script into your submissions directory e.g. in `algorithmic-efficiency/my_submissions`.
+2. Implement at least the methods in the submission_template. Feel free to use helper functions and/or modules as you see fit.
+3. Add a tuning configuration e.g. `tuning_search_space.json` file to your submission directory. For the tuning search space you can either:
+    1. Define the set of feasible points by defining a value for "feasible_points" for the hyperparameters:
+        ```
+        {
+            "learning_rate": {
+                "feasible_points": 0.999
+                },
+        }
+        ```
+        For a complete example see [tuning_search_space.json](https://github.com/mlcommons/algorithmic-efficiency/blob/main/reference_algorithms/target_setting_algorithms/imagenet_resnet/tuning_search_space.json).
+
+    2. Define a range of values for halton sampling by specifing a "min", "max" and "scaling" 
+        keys for the hyperparameter:
+        ```
+        {
+            "weight_decay": {
+                "min": 5e-3, "max": 1.0, "scaling": "log"
+                }
+        }
+        ```
+        For a complete example see[tuning_search_space.json](https://github.com/mlcommons/algorithmic-efficiency/blob/main/baselines/nadamw/tuning_search_space.json). 
+
+
+## Run your submission
+TODO(kasimbeg): Add script to run all workloads?
+
+You can evaluate your submission with the `submission_runner.py` module on one workload at a time. 
+To run 
+
+### JAX submissions
+To score your submission on a workload, from the algorithmic-efficency directory run: 
+```bash
+python3 submission_runner.py \
+    --framework=jax \
+    --workload=mnist \
+    --experiment_dir=<path_to_experiment_dir>\
+    --experiment_name=<experiment_name> \
+    --submission_path=<path_to_submission_module> \
+    --tuning_search_space=<path_to_tuning_search_space>
+```
+
+### PyTorch submissions
+To score your submission on a workload, from the algorithmic-efficency directory run: 
+```bash
+python3 submission_runner.py \
+    --framework=pytorch \
+    --workload=<workload> \
+    --experiment_dir=<path_to_experiment_dir> \
+    --experiment_name=<experiment_name> \
+    --submission_path=<path_to_submission_module> \
+    --tuning_search_space=<path_to_tuning_search_space>
+```
+
+#### PyTorch DDP
+We recommend using PyTorch's [Distributed Data Parallel (DDP)](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html) when using multiple GPUs on a single node. You can initialize ddp with torchrun. 
```

Comment:
I think we require it, right? So I would say "We use PyTorch's ..."

Reply:
I don't think it's "required" - the code supports running PyTorch workloads without DDP. I often run mnist & cifar workloads on my local machine with a single GPU.

Reply:
Wasn't sure if we strictly require it but I'll update the README as well.

Reply:
Sorry, missed Juhan's feedback. So I'll leave it then?

Reply:
sounds good, LGTM as is!